dom.go

```diff
@@ -104,16 +111,25 @@ type RenderSkipper interface {
 type HTML struct {
 	node jsObject
 
-	namespace, tag, text, innerHTML string
-	styles, dataset                 map[string]string
-	properties, attributes          map[string]interface{}
-	eventListeners                  []*EventListener
-	children                        []ComponentOrHTML
+	namespace, tag, text, innerHTML, key string
+	styles, dataset                      map[string]string
+	properties, attributes               map[string]interface{}
+	eventListeners                       []*EventListener
+	children                             []ComponentOrHTML
+	keyMap                               map[string]ComponentOrHTML
+	nestedKeys                           bool
+	insertBeforeNode                     jsObject
+	lastRenderedChild                    *HTML
 }
 
 // Node returns the underlying JavaScript Element or TextNode.
 func (h *HTML) Node() *js.Object { return h.node.(wrappedObject).j }
 
+// Key satisfies the Keyer interface.
```

Comment:
Replace `satisfies` with `implements`

Reply:
In Go terminology, interfaces are satisfied not implemented. However, it's good to keep the documentation consistent for this code base.

Reply:
I'm happy to be proven wrong, but I've never seen `<Method> satisfies the <Interface> interface.` over `<Method> implements the <Interface> interface.` in a docstring.

https://gotools.org/net/http#transport.go-L337 seems to agree with me.